### PR TITLE
dev, v3.0/tidb-in-kubernetes: tidb keep alive troubleshooting

### PR DESCRIPTION
**Pull request by cofyc**
### What is changed, added or deleted? 

Add troubleshooting notes about TCP keep alive.

### What is the related PR or file link(s)? 

https://github.com/pingcap/tidb-operator/issues/880
https://github.com/pingcap/tidb-operator/pull/915

### Which version does your change affect? 

dev
v3.0

### CODE REVIEW

File: dev/tidb-in-kubernetes/troubleshoot.md [hunk 1]
```diff
@@ -325,3 +325,20 @@ kubectl logs -f <tidb-pod-name> -n <namespace> -c tidb
         ```
 
     Pod 重建后，会以在集群中注册一个新的 Store，恢复完成。
+
+## TiDB 长连接被异常中断
+
+许多 Load Balancers 会设置连接空闲超时时间，当连接上没有数据传输超过设定值，将会主动将连接中断。若发现使用中，长查询会被异常中断，可检查客户端与 TiDB 服务端之间的中间件程序。若其连接空闲超时间较短，可尝试修改增大时间。若不可修改，可打开 TiDB `tcp-keep-alive` 选项，启用 TCP keepalive 特性。
```
Comment (yikeke) [inline dev/tidb-in-kubernetes/troubleshoot.md]:
```suggestion
许多负载均衡器 (Load Balancer) 会设置连接空闲超时时间。当连接上没有数据传输的时间超过设定值，负载均衡器会主动将连接中断。若发现 TiDB 使用过程中，长查询会被异常中断，可检查客户端与 TiDB 服务端之间的中间件程序。若其连接空闲超时时间较短，可尝试增大该超时时间。若不可修改，可打开 TiDB `tcp-keep-alive` 选项，启用 TCP keepalive 特性。
```
为了有助于翻译，补充了句子里的某些成分，请帮忙 double-check 补充的地方是否正确。

File: dev/tidb-in-kubernetes/troubleshoot.md [hunk 2]
```diff
@@ -325,3 +325,20 @@ kubectl logs -f <tidb-pod-name> -n <namespace> -c tidb
...
+
+Linux 默认发送 keepalive 探测包等待时间为 7200s 。若需减少，可通过 `podSecurityContext` 字段配置 sysctls ，例子如下：
```
Comment (yikeke) [inline dev/tidb-in-kubernetes/troubleshoot.md]:
```suggestion
默认情况下，Linux 发送 keepalive 探测包的等待时间为 7200 秒。若需减少，可通过 `podSecurityContext` 字段配置 `sysctls`，例子如下：
```

File: dev/tidb-in-kubernetes/troubleshoot.md [hunk 3]
```diff
@@ -325,3 +325,20 @@ kubectl logs -f <tidb-pod-name> -n <namespace> -c tidb
...
+```
+tidb:
+  ...
+  podSecurityContext:
+    sysctls:
+    - name: net.ipv4.tcp_keepalive_time
+      value: "300"
+```
+
+注：需要 tidb-operator 1.1+ 。
```
Comment (yikeke) [inline dev/tidb-in-kubernetes/troubleshoot.md]:
```suggestion
> **注意：**
>
> 以上配置的环境要求：TiDB Operator 1.1 版本或以上。
```

File: dev/tidb-in-kubernetes/troubleshoot.md (hunk 1 above)
Comment (cofyc) [reply dev/tidb-in-kubernetes/troubleshoot.md]: Thanks! Your suggested version is much better.

**Comment by cofyc** [comment]
hold this until https://github.com/pingcap/tidb-operator/pull/915 is merged.

**Pull request merged**